core/src/main/python/discover.py

```diff
@@ -189,20 +189,20 @@ def __process_variable_filename_arg(optional_arg_map):
     """
     _method_name = '__process_variable_filename_arg'
 
-    if CommandLineArgUtil.VARIABLE_PROPERTIES_FILE_SWITCH in optional_arg_map:
+    if CommandLineArgUtil.VARIABLE_FILE_SWITCH in optional_arg_map:
         variable_injector_file_name = variable_injector.get_default_variable_injector_file_name()
```

Comment:
the variable file is always the default, and never the name of the file provided on the command line switch